lib/apple/index.js

```diff
@@ -124,6 +124,11 @@ exports.verifyPayment = function (payment, cb) {
 		assert.equal(typeof payment.secret, 'string', 'Shared secret must be a string');
 		jsonData.password = payment.secret;
 	}
+	
+	if (payment.excludeOldTransactions !== undefined) {
+		assert.equal(typeof payment.excludeOldTransactions, 'boolean', 'Exclude old transactions must be a boolean');
```

Comment:
Sorry for nitpicking, but for clarity I would suggest:

`payment.excludeOldTransactions must be a boolean`

Reply:
Yeah agreed, I was just trying to be consistent with the shared secret error. I think we should change both

Reply:
Ah yeah, indeed 👍